autogen/main/versions.tf.tmpl

```diff
@@ -20,8 +20,10 @@ terraform {
   required_providers {
 {% if beta_cluster %}
     google-beta = ">= 3.42.0, <4.0.0"
+    kubernetes  = "~> 1.10, != 1.11.0"
 {% else %}
-    google = ">= 3.39.0, <4.0.0"
+    google     = ">= 3.39.0, <4.0.0"
+    kubernetes = "~> 1.10, != 1.11.0"
```

Comment:
Did this to pass the `terraform fmt` check - https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/774#issuecomment-752507037

There might be a way to keep things DRY here.